### ISSUE TITLE
Doc issue with copy buildings

### ISSUE THREAD

**Issue by Samspos** [MaFi-Games/Captain-of-Industry-issues#136]
i coppied the fluid docs all at once and only one was build prooperly, the other 3 need to be deconstructed and be build one at a time.
[doc issue.zip](https://github.com/MaFi-Games/Captain-of-Industry-issues/files/7973098/doc.issue.zip)

**Comment by MaFi-Games** [comment]
Yeah, copying of modular buildings is not properly fully supported.
